### ISSUE TITLE
'i' modifier is ambiguous false positive

### ISSUE THREAD

**Issue by Gamesh** [kalessil/phpinspectionsea#1402]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) |
| Language level | PHP 7.1                       |

Current behaviour
--
says 'i' modifier is ambiguous (no alphabet characters in a given pattern), but we do have a-z in it it's just a multiline regex, it probably does not check multiple lines

Expected behaviour
--
don't suggest removing i modifier

Example:
--
```
 private const REGEX = '/^
        (?P<id>\d+)                         # Numeric ID
        (?P<segment>[a-z]{1})               # Segment string letter
        (?P<revision>\d+)?                  # Revision optional if 0
    $/iDx';
```

**Comment by kalessil** [comment]
Looks like a bug, thank you for reporting @Gamesh.

**Comment by kalessil** [comment]
Fixed!

**Comment by Gamesh** [comment]
thank you
